### PR TITLE
Regeneration of the nexus file for fixing the changes coming from old…

### PR DESCRIPTION
**Pull request by RubelMozumder**
Regeneration of nxdl.

### CODE REVIEW

**Comment by lukaspie** [comment]
LGTM

File: base_classes/NXdetector.nxdl.xml [hunk 1]
```diff
@@ -470,6 +462,8 @@
             </doc>
             <!--. For a pixel detector it provides the nominal wavelength-->
             <!--. For other detectors this field has to be seen together with-->
+            <!--. For a pixel detector it provides the nominal wavelength-->
```
Comment (sanbrock) [inline base_classes/NXdetector.nxdl.xml:465]: is the comment duplicated?

**Review by sanbrock — COMMENTED** [review]
Licenses look correct, but NXsample and NXenvironment related restructuring is not yet in place

**Review by sanbrock — CHANGES_REQUESTED** [review]
why comment is duplicated?

File: base_classes/NXdetector.nxdl.xml (hunk 1 above)
Comment (RubelMozumder) [reply base_classes/NXdetector.nxdl.xml]: @sanbrock, There should not be any comments at all, this comments are coming from doc exactly on top of the comment blocks, where the order of the paragraphs was addressed by '#'. Therefore the paragraphs have been considered as comments by 'nyaml2nxdl' converter.

I fixed it by removing '#' by '1, 2, ...' from the order of paragraphs.

**Comment by RubelMozumder** [comment]
> Licenses look correct, but NXsample and NXenvironment related restructuring is not yet in place

The hierarchy is preserved perfectly for all, especially for NXsample and NXenvirionment. 
The main changes are in the design of XML file, not change in hierarchy or in text.

**Pull request merged**